tests/sles4sap/netweaver_test_instance.pm

```diff
@@ -26,6 +26,7 @@ sub run {
     # The SAP Admin was set in sles4sap/netweaver_install
     $self->set_sap_info(get_required_var('INSTANCE_SID'), get_required_var('INSTANCE_ID'));
     $self->become_sapadm;
+    $self->test_pids_max;
```

Comment:
Great! Could you also add this in `hana_test.pm`? With your function we can now easily test on both NW and HANA (to be sure that at least 2 SAP products are working).

Reply:
Done.